guide/youbot.md

```diff
@@ -0,0 +1,31 @@
+## KUKA's youBot
+
+%figure "youBot model in Webots"
+
+![model.png](images/robots/youbot/model.png)
+
+%end
+
+The youBot is a mobile robotic arm developed by [KUKA](http://www.youbot-store.com/).
+Its arm has five DOF and a linear gripper.
+Its base has four omnidirectional wheels allowing for omnidirectional movement.
```

Comment:
The first "ominidirectional" should be replaced by "Mecanum" and link to https://en.wikipedia.org/wiki/Mecanum_wheel